queries.sql

```diff
@@ -79,7 +79,47 @@ GROUP BY
     species;
 
 
+SELECT animals.name
+FROM animals
+JOIN owners
+ON animals.owners_id = owners.id
+WHERE owners.full_name = 'Melody Pond';
+
+
+SELECT animals.name
+FROM animals
+JOIN species ON animals.species_id = species.id
+WHERE species.species_name = 'Pokemon';
 
 
+SELECT owners.full_name, animals.name
+FROM owners
+LEFT JOIN animals ON owners.id = animals.owners_id;
 
 
+SELECT species.species_name, COUNT(animals.id) AS animal_count
+FROM species
+LEFT JOIN animals ON species.id = animals.species_id
+GROUP BY species.species_name;
+
+
+SELECT owners.full_name, COUNT(*) AS Digimon_count
+FROM owners
+JOIN animals ON owners.id = animals.owners_id
+JOIN species ON animals.species_id = species.id
+WHERE owners.full_name = 'Jennifer Orwell'
+  AND species.species_name = 'Digimon'
+GROUP BY owners.full_name;
```

Comment:
- [ ] I think Jennifer does have one. You might want to check this out :point_down: 

|Your Query| Requirement in Data|
|:--------------:|:-----------------:|
|![image](https://user-images.githubusercontent.com/99715640/271101774-061d2ec0-9df6-4502-a270-98030e0b31f8.png)|![image](https://user-images.githubusercontent.com/99715640/271101889-becee306-4f90-439e-a9e5-93fe4e8c48cd.png)|